core/src/test/java/com/exadel/etoolbox/linkinspector/core/services/resolvers/ExternalResultResolverImplTest.java

```diff
@@ -107,9 +111,10 @@ void testCheckLink_returnStatusCode200when404Head200Get() throws IOException {
     void testCheckLink_returnStatusCode200when200Head() {
         context.registerInjectActivateService(
                 new MockHttpClientBuilderFactory(),
-                ImmutableMap.of(
-                        MockHttpClientBuilderFactory.PN_STATUS_CODE, HttpStatus.SC_OK,
-                        MockHttpClientBuilderFactory.PN_STATUS_MESSAGE, "OK"));
+                Collections.unmodifiableMap(new HashMap<String, Object>() {{
```

Comment:
See the comment on unmodifiable map above

Reply:
done